dymension/validators/bridge/README_bridge_kaspa.md

```diff
@@ -66,6 +79,17 @@ Populate `.chains.<kaspa-network-name>.kaspaEscrowPrivateKey` with the escrow se
 }
 ```
 
+additionally, populate the following placeholders:
+
+| Placeholder                                                          | Description                                | Example                         |
+| -------------------------------------------------------------------- | ------------------------------------------ | ------------------------------- |
+| `<kaspa-network-rpc-url-without-protocol>`                           | Kaspa network wRPC URL without protocol    | `wrpc-kaspa.example.com`        |
+| `<kaspa-network-rest-url>`                                           | Kaspa network REST API URL                 | `https://api-kaspa.example.com` |
+| `<validator_escrow_secret>`                                          | Validator escrow private key (keep quotes) | `"your-private-key-here"`       |
+| `<dymension-hub-grpc-url>`                                           | Dymension hub gRPC URL                     | `https://grpc.example.com`      |
+| `<port>`                                                             | Respectful port number of the service      | `443`                           |
+| `<validator_ism_priv_key>`                                           | Validator ISM private key                  | `0xabcd1234...`                 |
```

Comment:
I think it should just be a table of field paths and description and example
not this placeholder thing, its more confusing